### PR TITLE
Add all refactored game files

### PR DESCRIPTION
**Pull request by beomjin-lee**
* Add all refactored game files

### CODE REVIEW

File: omok/main.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import configparser
```
Comment (beomjin-lee) [inline omok/main.py:1]: maybe this should be outside the pkg `omok`

**Review by beomjin-lee — COMMENTED** [review]
I think `main.py` should be outside the pkg, but also we should double-check against other Python libraries.

File: omok/main.py (hunk 1 above)
Comment (beomjin-lee) [reply omok/main.py:1]: Maybe not - https://stackoverflow.com/questions/193161/what-is-the-best-project-structure-for-a-python-application.

**Review by moonwonlee — APPROVED** [review]
Looks good to me

**Review by moonwonlee — APPROVED** [review]
LGTM

**Pull request merged**